examples/api/service/readme.md

```diff
@@ -28,8 +28,8 @@ Set up an ipfsnode.
 ```
 func main() {
 	// Basic ipfsnode setup
-	r := fsrepo.At("~/.ipfs")
-	if err := r.Open(); err != nil {
+	r, err := fsrepo.Open("~/.ipfs")
+	if err != nil {
```

Comment:
thanks! 

wonder if we ought to try and make our examples work as tests somehow. (i like how go does it-- "examples" are just code that is runnable, and that godoc treats specially.) this is probably unrealistic for anything complicated though...